src/rust/engine/process_executor/bazel_protos/src/.gitignore

```diff
@@ -0,0 +1,3 @@
+remote_execution.rs
+remote_execution_grpc.rs
+status.rs
```

Comment:
If `lib.rs` is always supposed to be the only live file, you can do something like:
```
*.rs
!lib.rs
```
...to ignore all non-`lib.rs` rs files.

Reply:
Nice :) Done